src/FEValues/face_values.jl

```diff
@@ -255,20 +256,22 @@ function BCValues(::Type{T}, func_interpol::Interpolation{dim,refshape}, geom_in
     end
 
     n_faces = length(qrs)
-    n_qpoints = n_faces == 0 ? 0 : length(getweights(qrs[1])) # assume same in all
+    n_qpoints = n_faces == 0 ? 0 : maximum(length.(getweights.(qrs))) # Bound number of qps correctly.
```

Comment:
```suggestion
    n_qpoints = n_faces == 0 ? 0 : maximum(qr->length(getweights(qr)), qrs) # Bound number of qps correctly.
```

Reply:
Sorry, I am not seeing the error. Can you elaborate?

Reply:
No error and nothing major, it just avoids allocating to temp arrays. 
```julia
using Ferrite, BenchmarkTools
julia> T=Float64; dim=3; refshape=RefCube; func_interpol=Lagrange{dim,refshape,1}(); geom_interpol=func_interpol;
julia> interpolation_coords = Ferrite.reference_coordinates(func_interpol);
julia> for face in Ferrite.faces(func_interpol)
               dofcoords = Vec{dim,T}[]
               for facedof in face
                   push!(dofcoords, interpolation_coords[facedof])
               end
               qrf = QuadratureRule{dim,refshape,T}(fill(T(NaN), length(dofcoords)), dofcoords) # weights will not be used
               push!(qrs, qrf)
           end
julia> getqpoints1(qrs) = maximum(length.(Ferrite.getweights.(qrs)));
julia> getqpoints2(qrs) = maximum(qr->length(Ferrite.getweights(qr)), qrs);
julia> @btime getqpoints1($qrs)
  29.246 ns (1 allocation: 160 bytes)
4
julia> @btime getqpoints2($qrs)
  5.600 ns (0 allocations: 0 bytes)
4
```